ibc-primitives/Cargo.toml

```diff
@@ -32,7 +32,8 @@ time        = { version = ">=0.3.0, <0.3.37", default-features = false }
 ibc-proto = { workspace = true }
 
 # cosmos dependencies
-tendermint = { workspace = true }
+tendermint       = { workspace = true }
+tendermint-proto = { workspace = true }
```

Comment:
We can remove `tendermint-proto` since it's not being used anywhere. Even if we plan to use it later, it's better to avoid adding this dependency given our goal of moving away from `tendermint` altogether.

Reply:
[531587c](https://github.com/cosmos/ibc-rs/pull/1325/commits/531587c4394909c3f0be2721b577d8f34a578d75)